components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/util/FrameworkConstants.java

```diff
@@ -700,4 +702,13 @@ public static class ActionIDs {
             public static final String PROCESS_CLAIM_CONSENT = "process-claim-consent";
         }
     }
+
+    /**
+     * Enum for authenticator param type.
+     */
+    public enum AuthenticatorParamType {
+
+        STRING,
+        INTEGER,
```

Comment:
what about boolean, double etc?

Reply:
Added these on need basis. Will add those and others when onboarding the relevant authenticators